resources/views/omnomcom/minisite.blade.php

```diff
@@ -6,7 +6,7 @@
 <html lang="en">
     <head>
         <link
-            href="https://fonts.googleapis.com/css?family=Roboto+Slab:400,700"
+            href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap"
```

Comment:
I think the inclusion here should be enough, then we do not need it in stylesheets.blade.php right?